dev/PagerControl/readme.md

```diff
@@ -0,0 +1,75 @@
+# PagerControl
+
+PagerControl allows developers to show items by providing an easy way to handle the user interface of pagination.
```

Comment:
Is there a better name for the file than readme? DevDesign maybe?

Reply:
The idea behind naming it readme.md is that GitHub will simply show the file when you open the PagerControl folder, just like the readme of NavigationView or the overall project readme.md file.